feature/more/src/main/java/com/epmedu/animeal/more/MoreViewModel.kt

```diff
@@ -1,15 +1,31 @@
 package com.epmedu.animeal.more
 
-import android.util.Log
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.epmedu.animeal.common.data.repository.ProfileRepository
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.SharedFlow
+import kotlinx.coroutines.flow.asSharedFlow
+import kotlinx.coroutines.launch
+import javax.inject.Inject
 
-internal class MoreViewModel : ViewModel() {
+@HiltViewModel
+internal class MoreViewModel @Inject constructor(
+    private val profileRepository: ProfileRepository
+) : ViewModel() {
+
+    private val _event = MutableSharedFlow<Event>()
```

Comment:
What do you think if we add `MutableSharedFlow` into `StateViewModel` for handling events?

cc: @Fileek 

Will be something like this

<img width="579" alt="image" src="https://user-images.githubusercontent.com/16294951/184147619-26c6a7bb-4036-4bf5-a4e4-8ba2f1e064ea.png">

<img width="853" alt="image" src="https://user-images.githubusercontent.com/16294951/184147656-fd139762-7313-4004-980c-edf9527b175f.png">

Reply:
Updated